TheraLang.Web/ClientApp/src/app/app.component.ts

```diff
@@ -1,18 +1,18 @@
-import { Component } from '@angular/core';
-import {TranslateService} from '@ngx-translate/core';
+import { Component } from "@angular/core";
+import { TranslateService } from "@ngx-translate/core";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.less']
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.less"]
 })
 export class AppComponent {
-  title = 'UTTMM';
+  title = "UTTMM";
   constructor(translate: TranslateService) {
     // this language will be used as a fallback when a translation isn't found in the current language
-    translate.setDefaultLang('uk');
+    translate.setDefaultLang("uk");
 
     // the lang to use, if the lang isn't available, it will use the current loader to get them
-    translate.use('uk');
+    translate.use("uk");
```

Comment:
maybe change "uk" to "ua" ?

Reply:
I'm assuming it refers to file which contains translations, which is still "uk.json". You've only changed the UI text.